### ISSUE TITLE
[Bug] 输入BUG标题 /mj are not working

### ISSUE THREAD

**Issue by deprecated-guy** [Licoy/ChatGPT-Midjourney#94]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Deployment**
- [ *] Docker
- [ ] Vercel
- [ ] Railway
- [ ] Server

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional Logs**
Add any logs about the problem here.

**Comment by Licoy** [comment]
you need input : /mj [prompt]

**Comment by yaohuiwu** [comment]
@Licoy You are right. This is a useful information but the author don't mention it anywhere. The doc is so bad almost make me crazy.
![image](https://github.com/Licoy/ChatGPT-Midjourney/assets/7850789/65376481-c773-4679-8126-34b34d12aab9)
